Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake

```diff
@@ -72,13 +72,24 @@ if(CGAL_DISABLE_GMP)
 endif()
 
 if(CGAL_DISABLE_GMP)
-  message(STATUS "GMP is disable. Try to use LEDA instead.")
-  set(GMPZ_NT ${LEDA_INT_NT})
-  set(QUOTIENT_CGAL_GMPZ_NT ${LEDA_RAT_NT})
-  set(CGAL_GMPQ_NT ${LEDA_RAT_NT})
-  set(LAZY_CGAL_GMPQ_NT ${LAZY_LEDA_RAT_NT})
-  set(LAZY_GMPZ_NT ${LAZY_LEDA_RAT_NT})
-  set(CGAL_GMPZ_NT ${LEDA_INT_NT})
+  message(STATUS "GMP is disable.")
+  if (CGAL_USE_LEDA)
+    message(STATUS "Try to use LEDA instead.")
```

Comment:
`"Try" -> `"  Try` as message is an explanation of the previous message.
(also acounts for the `MP case).